src/orchard/inspect.clj

```diff
@@ -408,17 +408,31 @@
           inspector
           mappable))
 
+(defn- left-pad
+  "TODO: should extract this into a library."
```

Comment:
[oh no!](https://www.theverge.com/2016/3/24/11300840/how-an-irate-developer-briefly-broke-javascript) 😄

Looks like `(String/format "%10s" (object-array ["foo"]))` ensures a given string has N characters, left-padding with spaces otherwise.

Reply:
Interesting, I've tried googling for it and found answers that String.format can only pad with zeros.

Still, I feel a bit reluctant to construct a format string and then use the slow formatting method for each iteration, seems a bit wasteful. Even though it will never be used for that many items anyway.

Yeah, it's probably nothing.